pkg/controller/mizar/mizar-arktos-network-controller.go

```diff
@@ -204,5 +261,149 @@ func (c *MizarArktosNetworkController) processNetworkCreation(network *v1.Networ
 	}
 
 	c.recorder.Eventf(network, corev1.EventTypeNormal, "processNetworkCreation", "successfully created network from mizar cluster: %v.", context)
+
+	// Create default VPC and Subnet after non-system tenant's arktos network is created successfully
+	klog.V(4).Infof("Mizar-Arktos-Network-controller - for non-system tenant (%s): start to create VPC(%s) and Subnet(%s)", network.Tenant, vpc, subnet)
+	err = createVpcAndSubnetCRD(network.Tenant, vpc, subnet, vpcDefaultTemplatePath, subnetDefaultTemplatePath, c.discoveryClient, c.dynamicClient)
```

Comment:
Why need another round of createVpcAndSubnetCRD for none system tenant? Shouldn't they use the same code?

Reply:
Default vpc and subnet for system tenant uses different codes from those for non-system tenant.  The trick was found in creating non-system tenant. 

The line 227 through line 235 are for system tenant whereas line 265 through line 272 are for non-system tenant, which are added later in testing non-system tenant.

Reply:
System tenant's network system-default-network has been in "Ready" status when the code are running line 227 whereas non-system tenant's network {{.Tenant}}-default-network is not in "Ready" status when code is running line 227, it needs Mizar to create it and ensure it is created successfully with successful event at line 263.